packages/mui-material/src/List/List.js

```diff
@@ -129,6 +137,14 @@ List.propTypes /* remove-proptypes */ = {
     PropTypes.func,
     PropTypes.object,
   ]),
+  /**
+   * The variant to use. There is no variants available by default.
+   * @default 'default'
+   */
+  variant: PropTypes /* @typescript-to-proptypes-ignore */.oneOfType([
+    PropTypes.oneOf(['default']),
+    PropTypes.string,
+  ]),
```

Comment:
```suggestion
  variant: PropTypes /* @typescript-to-proptypes-ignore */.string
```

Reply:
Are we supposed to edit these auto-generated files manually?

Reply:
In this case, yes. It is an exception for props that can be augmented. It would be nice if we can automate this in the future.